cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=64, required=False)
+    description = serializers.CharField(max_length=255, required=False)
+    duration = serializers.IntegerField(required=False)
```

Comment:
title and duration are required in a model, but not required in serializer, that's strange

Reply:
Fixed